microcosm_flask/swagger/naming.py

```diff
@@ -21,11 +19,11 @@ def operation_name(operation, obj):
         foo.search_for.bar => client.foo.search_for_bars()
 
     """
-    if isinstance(obj, (list, tuple)):
-        verb, rest = operation.value.name, obj[1:]
+    verb = operation.value.name
+    if ns.object_:
```

Comment:
This is one of the few places that the internals of `Namespace` leak out. I wasn't ready to make the `Namespace` responsible for knowledge of Swagger naming conventions.